### PR TITLE
delete obsolete calc_damage endpoint and also pull out routing logic

### LINKED ISSUE #53: stop lying about gcd and mp consumption

### PR DESCRIPTION
**Pull request by ackao**
fixes #53 

also begins to address breakdown of routing/backend app logic #39

### CODE REVIEW

**Comment by ackao** [comment]
pls ignore code ugliness within routing functions (this was copy and paste)

File: backend/xivdpscalc/routing.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+from flask import Flask, jsonify, request, abort
```
Comment (redshadowhero) [inline backend/xivdpscalc/routing.py:1]: `Flask` is an unused import here

File: backend/xivdpscalc/routing.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+from flask_cors import CORS
```
Comment (redshadowhero) [inline backend/xivdpscalc/routing.py:2]: same thing – I don't think you're using this

**Review by redshadowhero — CHANGES_REQUESTED** [review]
Small changes at first pass – I didn't really look through the angular code at all.

**Pull request closed without merging**